.buildkite/pipeline.yml

```diff
@@ -17,10 +18,10 @@ steps:
       - make build_xcframework
       - make build_swift
       - make build_ios_static
+      - zip -r Bugsnag.xcframework.zip build/xcframeworks/products/Bugsnag.xcframework
```

Comment:
This should also have the `--symlinks` flag so that it doesn't actually store the symlinked files twice.